src/jumio-kyc/kyc.controller.ts

```diff
@@ -63,21 +63,16 @@ export class KycController {
   ): Promise<SerializedKyc | { can_attempt: boolean }> {
     const redemption = await this.redemptionService.find(user);
 
-    const canAttemptError = await this.redemptionService.canAttempt(
-      redemption,
-      user,
-    );
-
-    const canAttempt = !canAttemptError;
+    const eligibleError = await this.redemptionService.isEligible(user);
 
     if (!redemption) {
-      return { can_attempt: canAttempt };
+      return { can_attempt: !eligibleError };
```

Comment:
Same as other comments, might be good to return a message that can be passed to the UI